packages/transaction-controller/package.json

```diff
@@ -85,7 +85,7 @@
     "@babel/runtime": "^7.23.9",
     "@metamask/approval-controller": "^6.0.0",
     "@metamask/gas-fee-controller": "^15.0.0",
-    "@metamask/network-controller": "^18.1.2"
+    "@metamask/network-controller": "^18.0.0"
```

Comment:
I think for `transaction-controller`, we do want to ensure we use the same version of `@metamask/eth-block-tracker`? So I think for this specific package, we do want the bump in peerDependencies entry.

Reply:
Hmm. The way we've been using the peer dependency is that it dictates which version of this package the project _needs_ to use in order to be able to use this controller in all situations without breaking.

So are you saying that in order to use `transaction-controller` without breaking, a project needs to effectively rely on the version of `network-controller` that uses `@metamask/eth-block-tracker`?

Reply:
> So are you saying that in order to use `transaction-controller` without breaking, a project needs to effectively rely on the version of `network-controller` that uses `@metamask/eth-block-tracker`?

Yes, due to (or the other way around, depending on how you want to see the causality) the switch from `nonce-tracker` to `@metamask/nonce-tracker` and its associated switch from `eth-block-tracker` to `@metamask/eth-block-tracker`.